### ISSUE TITLE
Clock Time Problem

### ISSUE THREAD

**Issue by Giunillo** [LIJI32/SameBoy#276]
The ingame clock time is frozen while playing Pokemon G/S/C, and don't refresh properly even while Saving/Loading.
Tested on Windows 10, SDL version 0.13.3 , with different devices and roms.

**Comment by Giunillo** [comment]
NOTE: On Windows 10 SDL version 0.12.3 everything works fine.

**Comment by GeekyPlayzGH** [comment]
Still doesn't work for me, using Windows 10 Home Version 10.0.18363 Build 18363.

**Comment by NieDzejkob** [comment]
This got broken in
```
commit 0ed5cf6b3879c0b7d6adfe769354e55fe04d1d1b
Author: Lior Halphon <LIJI32@gmail.com>
Date:   Wed Apr 8 19:07:29 2020 +0300

    Proper MBC30 support, more accurate MBC3 emulation. Fixes #244
```

The handler introduced for MBC register writes now clamps the value to be in the range [0; 3], which means it's never in the [8; 0xC] required to read RTC data (cf. `Core/memory.c:98`, `Core/mbc.c:91`).

**Comment by LIJI32** [comment]
Duh, good catch. I'll get it fixed tomorrow.
